public/static/docs/api-reference/get_url.md

```diff
@@ -41,39 +50,31 @@ an appropriate library, such as
 or
 [`paramiko`](https://docs.paramiko.org/en/stable/api/sftp.html#paramiko.sftp_client.SFTPClient.get).
 
-Note that if the target is a directory, the URL will end in `.dir`, as DVC
-stores a special JSON file with `.dir` extension that contains the mapping of
-files in the directory (as a JSON array), along with their hash values. Refer to
-[Structure of cache directory](/doc/user-guide/dvc-files-and-directories#structure-of-cache-directory)
-and `dvc add` to learn more about how DVC handles data directories.
-
 ## Parameters
 
 - **`path`** - location and file name of the file or directory in `repo`,
   relative to the project's root.
 
 - `repo` - specifies the location of the DVC project. It can be a URL or a file
-  system path. Both HTTP and SSH protocols are supported for online Git repos
-  (e.g. `[user@]server:project.git`).
-
-  The current project is used by default if a `repo` argument is not given (the
-  current working directory tree is walked up to find it).
+  system path. Both HTTP and SSH protocols are supported for online Git
+  repositories (e.g. `[user@]server:project.git`). _Default_: The current
+  project is used (the current working directory tree is walked up to find it).
 
 - `rev` - Git commit (any [revision](https://git-scm.com/docs/revisions) such as
   a branch or tag name, or a commit hash). If `repo` is not a Git repo, this
-  option is ignored.
-
-  `HEAD` is used by default if a `rev` argument is not given.
+  option is ignored. _Default_: `HEAD`.
 
 - `remote` - name of the [DVC remote](/doc/command-reference/remote) to use to
-  form the returned URL string.
+  form the returned URL string. _Default_: The
+  [default remote](/doc/command-reference/remote/default) of `repo` is used.
 
-  The [default remote](/doc/command-reference/remote/default) of `repo` is used
-  if a `remote` argument is not given.
+## Exceptions
 
-  A `dvc.exceptions.NoRemoteError` is raised if no `remote` is found.
+- `dvc.api.UrlNotDvcRepoError` - `repo` is not a DVC project.
 
-## Example: Get the remote storage URL to a data file
+- `dvc.exceptions.NoRemoteError` - no `remote` is found.
```

Comment:
What's the motivation behind this new section? I like that it's structured and easy to find (via the right hand nav) but it's also good to have the exceptions in their context (where they were before, in the description).

Reply:
Yep, you can find them when you need them ... and you don't need them often ... it makes it easier to read what's important

Reply:
also, we need to check - there should be other relevant exception here

Reply:
Yep, please check this comment: https://github.com/iterative/dvc.org/pull/908#discussion_r386204040